pkg/vrraw/vrConsts_test.go

```diff
@@ -0,0 +1,181 @@
+package vrraw_test
```

Comment:
You know, I'm not sure if this needs to be tested in this way--this is just a definition of constants. The test isn't testing any functionality, it's just testing that constant values are set to...constant values. I would consider just dropping the test for this.

Reply:
I agree that looking at it after the fact, it seems a bit silly, but my thought process when testing it was that if I forced myself to re-type the expected values for a unit test, I would be more likely to catch typos in the actual values.

As it so happened, I did fat finger "UV" as "UC" or something, so the act of writing the test did catch that.

I would lean toward keeping it, if anyone ever goes in and makes changes, this becomes codegen in some way, or an IDE accidentally alters the string values of a const during a refactor of a local variable somewhere (I've had this happen), the unit tests will be useful to have.

That being said I am also happy to remove it.

Reply:
Your point is well-taken, though I'm still a bit on the fence about it. This is because the test doesn't really enforce any _functionality_, but merely is a second definition of the constants and enforce that both change together. Since these constants are meaningful, I could see a test where we maybe check that they are consistent with another implementation or some external reference or something, but an internal reference seems a bit odd to me. 

Setting a constant to an incorrect literal value seems like something that needs to be checked in a Code Review regardless of if the test exists or not. 

That being said, it doesn't really hurt, other than some additional overhead whenever a new VR is added. Since that's unlikely to be a common occurence, perhaps leaving the test is okay. Also, if we decide it's not needed, we can always easily remove it in a future refactor. 

I'd lean towards not adding it given the above, but in this case, I'll leave the final decision up to you and won't block the submission on it. It's a bit odd imo, but not actively harmful.

Reply:
I will remove them in this update.